tests/e2e/release/cht-config/generate-short-codes.wdio-spec.js

```diff
@@ -0,0 +1,75 @@
+const utils = require('../../../utils');
+const sUtils = require('../../sentinel/utils');
+const commonElements = require('../../../page-objects/common/common.wdio.page');
+const loginPage = require('../../../page-objects/login/login.wdio.page');
+const reportsPage = require('../../../page-objects/reports/reportsPage.wdio');
+const personFactory = require('../../../factories/cht/contacts/person');
+const place = require('../../../factories/cht/contacts/place');
+const places = place.generateHierarchy();
+const clinic = places.find((place) => place.type === 'clinic');
+
+const contact = personFactory.build(
+  {
+    parent: {
+      _id: clinic._id,
+      parent: clinic.parent
+    },
+    phone: '+254712345670'
+  });
+
+const docs = [...places, contact];
+
+describe('generating short codes', () => {
+  const submit =  (body) => {
```

Comment:
I feel this is a generic enough and potentially re-usable method which can be moved in the `util` file itself. Something like:
```
utils.performPost(path, postBody);
```